### PR TITLE
Fix #2 - Transformed to i18n ready Jekyll

### LINKED ISSUE #2: content management met jekyll
Just like the status page. This allows people to change the content by making PRs

### PR DESCRIPTION
**Pull request by bwbroersma**
- Transformed to i18n ready Jekyll
- Moved inline styles to css file
- Fixed HTML validation: closing div in footer
- Small README touch plus added build information

### CODE REVIEW

**Comment by arianvp** [comment]
:partying_face:

File: README.md [hunk 1]
```diff
@@ -3,42 +3,95 @@
...
 * Improve the onboarding for new community members (also for non-techies, see below)
-* Landing page for Github, Trello, Slack etc. Point out the main features of the community 
+* Landing page for Github, Trello, Slack etc. Point out the main features of the community
 * Give props to the most active community members
 * A showcase page for the openness of the project, that will hopefully inspire other government projects to be more open too
 * Set the right atmosphere and expectations for people that want to help out
 * Make it clear that this is partly a community driven project. Everyone is allowed to contribute, criticize, etc.
 
-## Tartget audience 
+## Tartget audience
```
Comment (arianvp) [inline README.md]:
```suggestion
## Target audience
```

**Comment by arianvp** [comment]
How exactly does the translation of the `_blocks` files work? (Or how would they work?)

**Comment by bwbroersma** [comment]
> How exactly does the translation of the `_blocks` files work? (Or how would they work?)

Same as in the [app status repo](https://github.com/minvws/nl-covid19-notification-app-status): copy the base (preferably keep the name with `-en` suffix), specify `lang: en` and translate all text and URLs. I tried to go over it in the README, you're welcome to improve the README on that point :slightly_smiling_face:

**Pull request merged**